pkg/report/table.go

```diff
@@ -19,6 +22,9 @@ type TableWriter struct {
 	Severities []dbTypes.Severity
 	Output     io.Writer
 
+	// We have to show a message once about using the '-format json' subcommand to get the full pkgPath
+	ShowMessageOnce *sync.Once
```

Comment:
Does it need to be a pointer? I expected something like this.

```suggestion
	showMessageOnce sync.Once
```